### DISCUSSION TITLE
Welcome to Swarm Discussions!

### DISCUSSION THREAD

**Discussion by RidhwanDev** [RidhwanDev/Swarm#1] [General]
## 👋 Welcome to Swarm!
  We’re using Discussions as a place to connect with other members of our community. We can use this space to
  * Discuss Ideas that may or may not already be on the roadmap
  * Suggest new ideas
  * Give feedback on how the experiment is going
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do.

**Comment by lexcraw4d** [comment]
I have some ideas I would love to talk about. I am Lexie a Full Stack Dev. I am dev driftwood in discord. Let's connect!

**Reply by RidhwanDev** [reply]
Nice to meet you Lexie! Would love to hear more about your ideas. Feel free to also drop them in discord until we have this properly set up :)
